packages/ui/packages/Loading/src/styles.js

```diff
@@ -10,6 +10,7 @@ export const StyledLoading = Styled.div`
     height: 0.7em;
     animation-fill-mode: both;
     animation: loading 1.8s infinite ease-in-out;
+    will-change: box-shadow;
```

Comment:
Important: will-change is intended to be used as a last resort, in order to try to deal with existing performance problems. It should not be used to anticipate performance problems.
https://developer.mozilla.org/en-US/docs/Web/CSS/will-change

Reply:
@MostafaRastegar well it is used as a last resort :) maybe we should change skeleton loading animation from opacity to background transform